---
layout: index
title: Home
sidebar_sort_order: 1
---

## Research interests

I'm broadly interested in the application of computation and mathematics to medicine, but currently in my postdoc I'm looking at how the variants that regulate transcript splicing can help us understand the pathways of inhereted disease risk.

## Previous projects

## PhD thesis: Genetic architecture of transcript splicing in blood and phenotypic consequences

For my doctorate in Mathematical Genomics and Medicine, in the lab of [Dr. Emma Davenport](https://davenportlab.com/) I studied the differences in transcript splicing between healthy individuals and those suffering sepsis. For both of these populations, I additionally described their genetic regulators of disease-context specific transcript splicing, and how those variants potentially modulate downstream phenotypic traits through a splicing mechanism. Finally, I described how these variants in sepsis could be used to understand the inherited risk of related immune conditions.

### Phylogenomic analysis of an infant gut commensal

For my second rotation project, I performed the largest to-date analysis of the infant probiotic bacteria *Bifidobacterium breve*. This was in the lab of [Trevor Lawley](https://www.sanger.ac.uk/group/lawley-lab/) at the [Wellcome Sanger Institute](https://www.sanger.ac.uk/). By looking at a collection of 414 genomes, I was able to contrast the metabolic potential of each, as well as make inferences about the distribution and capabilities of the species. I also demonstrated that existing studies which have only assessed in the order of 20-30 isolates are vastly under powered and neglect geographically diverse lineages which I correct for.

### Asthma transcription network analyses
In the lab of [Mike Inouye](http://www.inouyelab.org/){:target="_blank"} at the [Baker Heart and Diabetes Institute](https://baker.edu.au/){:target="_blank"} in Melbourne, Australia, I clustered gene expression networks to aid in analysing the development and progression of asthma in infants. This approach summarises large groups of similarly-regulated genes as clusters to simplify downstream analysis.

### Plasmid genomics
With [Kat Holt](https://holtlab.net/){:target="_blank"} at the [Bio21 Institute](https://www.bio21.unimelb.edu.au/){:target="_blank"} in Melbourne, Australia, I analysed the genetics of plasmids in bacteria using a network based approach.
This was based on the observation that even among closely related plasmids, their shared genetic content wildly differs, and can easily become mixed up with the genes of the host bacteria.
Scripts for running and visualising this yourself is available on [__GitHub__](https://github.com/atokolyi/PlasCliques){:target="_blank"}.

### Cardiac transcriptomics
In the lab of [Mirana Ramialison](https://www.armi.org.au/our-groups/ramialison-group/){:target="_blank"} at the [Australian Regenerative Medicine Institute](https://www.armi.org.au/){:target="_blank"} in Clayton, Australia, I explored how heart transcription data could be interactively visualised in 3D.
This visualisation can be run in a web browser, and is accessible online at [__3D Cardiomics__](http://3d-cardiomics.erc.monash.edu.au/){:target="_blank"}.
We also have a paper out in the [__Journal of Molecular and Cellular Cardiology__](https://doi.org/10.1016/j.yjmcc.2021.09.011){:target="_blank"}.

