---
layout: index
title: Home
---

## Research interests

I'm broadly interested in the application of computation and mathematics to biology, but at the moment I enjoy looking at the interactions between humans and microbes (e.g. bacteria, viruses, and fungi), and how these relate to health and disease.

## Current topics

At the moment I'm in my second of the 4-year Wellcome Trust PhD Programme in Mathematical Genomics and Medicine. Having previously done a rotation project in [Emma Davenport's lab](https://davenportlab.com/) at the Sanger Institute (blog post [here](/2020/05/08/end-of-first-rotation.html)), I then decided to make this my full PhD Project. I'll be focusing on human variation in gene transcription (such as alternative splicing) and how this contributes to the heterogeneity of immune responses. I'll also be looking at the interactions of different microbes (both pathogenic and commensal) on this.

*Check out my [__blog__](/blog/) for more details on these.*

## Previous projects

### Phylogenomic analysis of an infant gut commensal

For my second rotation project, I performed the largest to-date analysis of the infant probiotic bacteria *Bifidobacterium breve*. This was in the lab of [Trevor Lawley](https://www.sanger.ac.uk/group/lawley-lab/) at the [Wellcome Sanger Institute](https://www.sanger.ac.uk/). By looking at a collection of 414 genomes, I was able to contrast the metabolic potential of each, as well as make inferences about the distribution and capabilities of the species. I also demonstrated that existing studies which have only assessed in the order of 20-30 isolates are vastly under powered and neglect geographically diverse lineages which I correct for.

### Asthma transcription network analyses
In the lab of [Mike Inouye](http://www.inouyelab.org/){:target="_blank"} at the [Baker Heart and Diabetes Institute](https://baker.edu.au/){:target="_blank"} in Melbourne, Australia, I clustered gene expression networks to aid in analysing the development and progression of asthma in infants.
This approach summarises large groups of similarly-regulated genes as clusters to simplify downstream analysis.


### Plasmid genomics
With [Kat Holt](https://holtlab.net/){:target="_blank"} at the [Bio21 Institute](https://www.bio21.unimelb.edu.au/){:target="_blank"} in Melbourne, Australia, I analysed the genetics of plasmids in bacteria using a network based approach.
This was based on the observation that even among closely related plasmids, their shared genetic content wildly differs, and can easily become mixed up with the genes of the host bacteria.
Scripts for running and visualising this yourself is available on [__GitHub__](https://github.com/atokolyi/PlasCliques){:target="_blank"}.

### Cardiac transcriptomics
In the lab of [Mirana Ramialison](https://www.armi.org.au/our-groups/ramialison-group/){:target="_blank"} at the [Australian Regenerative Medicine Institute](https://www.armi.org.au/){:target="_blank"} in Clayton, Australia, I explored how heart transcription data could be interactively visualised in 3D.
This visualisation can be run in a web browser, and is accessible online at [__3D Cardiomics__](http://3d-cardiomics.erc.monash.edu.au/){:target="_blank"}.
We also have a paper out on [__bioRxiv__](https://www.biorxiv.org/content/10.1101/792002v1){:target="_blank"}.

