---
layout: post
title: My first PhD rotation project in RNA splicing in sepsis
---

Today I will be starting my first rotation project in the [Davenport Lab](http://davenportlab.com/){:target="_blank"} at the [Wellcome Sanger Institute](https://www.sanger.ac.uk/){:target="_blank"}. I'll be looking at how RNA transcripts are differentially spliced between sepsis patients with different infections and immune responses. Hopefully, this will give some insight into the underlying regulation of this condition, with the aim of improving treatment.

### Difficulties in the study of sepsis

Sepsis is currently recognised to manifest as life-threatening organ dysfunction as a result of
a dysregulated host response to infection. In contrast to a well tolerated infection, in sepsis the host response is disproportionate, non-homeostatic, and injures the organs and tissues of the host.  Challenges in the study of this condition and hence the stratification of individuals for research purposes are caused by wildly varying causes of infection; with bacteria, viruses and fungi all commonly observed, in addition to the heterogeneity of the host immune response. As such, there is currently no simple and unambiguous test that identifies a sufferer of sepsis.

### How gene splicing is involved

Studies of gene expression have previously been undertaken to determine the regulatory networks of the immune system, with the goal of understanding what goes wrong in people with harmful responses. Though, when the full transcript is sequenced with technology such as RNA-seq, we can glean transcript level information unavailable with microarrays, such as alternative transcription start and end sites, exonic mutations, as well as intron usage and quantification through splicing analyses. I will be looking at comparing intron usage between individuals with [Leafcutter](https://davidaknowles.github.io/leafcutter/){:target="_blank"}. A major benefit of leafcutter is that it is annotation-free, allowing the discovery of novel isoforms and exon-spanning junctions. This makes it a justified choice in uncovering differentially spliced variants and sQTLs in a disease cohort.
